src/lib/es2020.bigint.d.ts

```diff
@@ -1,3 +1,103 @@
+type From1To21 = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21;
+type From0To20 = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20;
+
+interface BigIntToLocaleStringOptionsBase {
+    /**
+     * The locale matching algorithm to use.The default is "best fit". For information about this option, see the {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation Intl page}.
+     */
+    localeMatcher?: "lookup" | "best fit" | string;

```

Comment:
`"x" | string` is currently reduced to `string` and the literals are thrown out. I think `string` is fine for now.

Reply:
The feature request for supporting this is https://github.com/microsoft/TypeScript/issues/33471

Reply:
Amazing! Why I remembered I have used this feature in somewhere when there is even not a PR!
Do I need to remove these "x", or just leave them here, waiting for the benefit of this feature request?